src/extension/index.ts

```diff
@@ -164,203 +148,93 @@ export async function activate(context: vscode.ExtensionContext) {
     )
   );
 
-  registerCommand(context, "neo3-visual-devtracker.neo.newContract", () =>
-    Templates.newContract(context)
-  );
+  registerCommand(context, "neo3-visual-devtracker.neo.newContract", () => Templates.newContract(context));
 
-  registerCommand(context, "neo3-visual-devtracker.neo.walletCreate", () =>
-    NeoCommands.createWallet()
-  );
+  registerCommand(context, "neo3-visual-devtracker.neo.walletCreate", () => NeoCommands.createWallet());
 
-  registerCommand(context, "neo3-visual-devtracker.connect", () =>
-    activeConnection.connect()
-  );
+  registerCommand(context, "neo3-visual-devtracker.connect", () => activeConnection.connect());
 
-  registerCommand(context, "neo3-visual-devtracker.customizeServerList", () =>
-    serverListDetector.customize()
-  );
+  registerCommand(context, "neo3-visual-devtracker.customizeServerList", () => serverListDetector.customize());
 
-  registerCommand(context, "neo3-visual-devtracker.disconnect", () =>
-    activeConnection.disconnect()
-  );
+  registerCommand(context, "neo3-visual-devtracker.disconnect", () => activeConnection.disconnect());
 
-  registerCommand(
-    context,
-    "neo3-visual-devtracker.express.contractDeploy",
-    (commandArguments) =>
-      NeoExpressCommands.contractDeploy(
-        neoExpress,
-        contractDetector,
-        blockchainsTreeDataProvider,
-        commandArguments
-      )
+  registerCommand(context, "neo3-visual-devtracker.express.install", () => NeoExpressCommands.install(context));
```

Comment:
I hadn't thought about having an explicit command for installing/updating Express. Nice addition